catalog/ui/catalog-ui-search/src/main/webapp/js/model/User.js

```diff
@@ -142,7 +142,8 @@ define([
                 columnHide: [],
                 columnOrder: ['title', 'created', 'modified', 'thumbnail'],
                 uploads: [],
-                fontSize: '16'
+                fontSize: '16',
```

Comment:
We should consolidate handling of the resultCount here to ensure it is never undefined / greater than the admin setting.